tests/game.test.js

```diff
@@ -0,0 +1,33 @@
+const Game = require('../game');
+
+describe('Game', () => {
+  const game = new Game();
+
+  it('plays 10 frames', () => {
+    expect(game.frames.length).toEqual(10);
+  })
+
+  it('can record a roll', () => {
+    game.roll(1);
```

Comment:
is an expect line needed here? Or is it implied that if game.roll(1) doesn't throw an error, that the roll is recorded?

Reply:
I just forgot to put a test - whoops